docs/contributor/busola-docker.md

```diff
@@ -0,0 +1,24 @@
+## Busola in Docker
+
+### Adding a Cluster Using Kubeconfig ID
+
+1. If you run Busola in Docker, you can mount your kubeconfig as a bind mount for Busola container. Execute the following command:
+
+   ```bash
+   docker run --rm -it -p 3001:3001 -v <path to your kubeconfig>:/app/core-ui/kubeconfig/<your kubeconfig file name> --pid=host --name busola europe-docker.pkg.dev/kyma-project/prod/busola:latest
```

Comment:
instead of these `<path to foobar>` please use envvar notation: `$kubeconfig`. This enables the reader to copy-paste-run the command with the envvars set properly, without having to edit the command line at all.